assets/training/model_management/environments/model-management/context/requirements.txt

```diff
@@ -13,5 +13,4 @@ azureml-telemetry=={{latest-pypi-version}}
 azureml-automl-dnn-vision=={{latest-pypi-version}}
 pyarrow==14.0.2
 GitPython=={{latest-pypi-version}}
-paramiko=={{latest-pypi-version}}
-tqdm=={{latest-pypi-version}}
\ No newline at end of file
+paramiko=={{latest-pypi-version}}
```

Comment:
Could you please share, how is removing tqdm from here and adding in dockerfile helpful?